### DISCUSSION TITLE
Welcome to 03-rsvd-team-3 Discussions!

### DISCUSSION THREAD

**Discussion by Peng-Rao** [AMSC-24-25/03-rsvd-team-3#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DOCCA0** [comment]
### Tasks:
1. Implement rSVD exploiting PM and compare with numpy.
2. Implement rSVD exploiting QR with Givens rotations and compare with numpy.
3. PCA
4. DMD
5. Image compression
6. POD

### Q:
1. Implement PCA+DMD+img compression+POD or just some of them?
2. The last 4 tasks depends on the first 2 tasks
